src/poliastro/threebody/soi.py

```diff
@@ -0,0 +1,108 @@
+"""Sphere of Influence
+
+Contains methods to compute radius of sphere of influence.
+
+Laplace Radius: A laplace sphere of influence (SOI) in astrodynamics and astronomy is the oblate-spheroid-shaped
+region around a celestial body where the primary gravitational influence on an orbiting object is that body.
+This is usually used to describe the areas in the Solar System where planets dominate the orbits of
+surrounding objects such as moons, despite the presence of the much more massive but distant Sun. In the
+patched conic approximation, used in estimating the trajectories of bodies moving between the neighbourhoods
+of different masses using a two body approximation, ellipses and hyperbolae, the laplace radius is taken as
+the boundary where the trajectory switches which mass field it is influenced by.
+The result is:
+
+.. math::
```

Comment:
I have followed this : http://openalea.gforge.inria.fr/doc/openalea/doc/_build/html/source/sphinx/rest_syntax.html#maths-and-equations-with-latex

I am still getting some issues

Reply:
> I am still getting some issues

The issue is very explicit, I copy pasted it in another comment. Before pushing, try to `make html` locally to make sure it works. Or even better, use the command that CircleCI uses to turn warnings into errors.